.gitignore

```diff
@@ -58,4 +58,4 @@ package-lock.json
 static_build
 static_final
 static
-npm-debug.log
+npm-debug.log
```

Comment:
`.gitignore` needs a blank line at the end of the file :)

Reply:
fixed